src/System.ComponentModel.Annotations/tests/PhoneAttributeTests.cs

```diff
@@ -24,6 +24,21 @@ protected override IEnumerable<TestCase> ValidValues()
             yield return new TestCase(new PhoneAttribute(), "425-555-1212 ext.123");
             yield return new TestCase(new PhoneAttribute(), "425-555-1212 ext. 123");
             yield return new TestCase(new PhoneAttribute(), "1");
+
+            // Certain invalid phone numbers are reported as valid with .NET core.
+            // The full .NET framework considers them invalid. This is likely a bug
+            // in .NET core.
+            if (!PlatformDetection.IsFullFramework)
```

Comment:
What I meant was to disable the test that is failing in netcoreapp due to bug: https://github.com/dotnet/corefx/issues/17873

Instead of making this test pass, as with doing that we would be testing a buggy behavior. So we should add [ActiveIssue(17873)] to the netcoreapp test itself instead of hacking the input to have the "expected" behavior which we claim it to be a bug.

Another approach would be skipping the test in netcoreapp and running it on netfx so that we don't loose test coverage in Desktop.